.travis.yml

```diff
@@ -1,16 +1,10 @@
-language: cpp
-sudo: required
-
+language: generic
 services: docker
 
 before_install:
- - docker pull ubuntu
- - docker run --cidfile /tmp/id -it -d ubuntu /bin/bash
+ - docker run --cidfile /tmp/id -it -d anselmolsm/travis_ci_deps:ubuntu1604 /bin/bash
```

Comment:
I actually think it's better to be explicit about the env setup and leave it here rather than relying on a third party image containing everything installed.  This way it's easy to keep track of the packages that need to be installed.

If it's in fact much longer to install the packages from scratch, we should provide a Dockerfile to replicate the build environment from scratch.